README.md

```diff
@@ -1,9 +1,20 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+In order to connect to the two databases successfully, a developer must add in a connection.js file (which creates a connection to the server and sets PGDATABASE to either test or development depending on whether it is run within the Jest test suite or not).
 
+For the test database, an app.test.js file is required where SuperTest is used to listen to the server when running the tests, and the database can be re-seeded between each test to ensure the tests are ran consistently and changes to the data aren't persistent (whilst the smaller database is also more manageable).
 
+For the development database, a listen.js file is required to ensure that the server is listening when any requests are made to ensure it can send an appropriate response.
```

Comment:
You are definitely right that anyone who wants to use your repo needs these files! However, you would be expected to include this in the repo, rather than ask whoever clones it to create their own. For example, I can see that you already have a connection.js file in your repo! I'd suggest removing these first few sentences. The information in your next section is all that a developer would need to run the repo in it's current form.